src/screens/Customers/CustomerMain.tsx

```diff
@@ -1,228 +1,243 @@
+      <AttachMoneyIcon fontSize='inherit' className={classes.tabIcon} />
+      <Typography className={classes.tabLabel}>Payment</Typography>
+    </div>
+  );
+
   return (
-    <BaseScreen rightIcon="user">
-      <div className={classes.headerWrapper}>
-        <h1 className={classes.title}>Customers</h1>
-      </div>
-      <div className={classes.selectionHeader}>
-        <SearchBar placeholder="Search for a customer" onSearchChange={handleSearchChange} />
-        <FormControl>
-          <Select onChange={handleMenuSelect} multiple value={sortAndFilter} inputProps={{ 'aria-label': 'Without label' }}>
-            <ListSubheader>Sort By</ListSubheader>
-            <MenuItem value="NAME">{SortBy.NAME}</MenuItem>
-            <MenuItem value="METER">{SortBy.METER}</MenuItem>
-            <ListSubheader>Filter By</ListSubheader>
-            <MenuItem value="PAYMENT_STATUS">{FilterBy.PAYMENT_STATUS}</MenuItem>
-            <MenuItem value="METER_STATUS">{FilterBy.METER_STATUS}</MenuItem>
-            <MenuItem value="ACTIVE_STATUS">{FilterBy.ACTIVE_STATUS}</MenuItem>
-          </Select>
-          <div className={classes.rightAlign}><FormHelperText>Sort and Filter</FormHelperText></div>
-        </FormControl>
-      </div>
-      <BaseScrollView>
-        <FormHelperText>{filterLabels[0]}</FormHelperText>
-        {filteredCustomers.map((customer, index) => (
-          <Link key={index} to={`${props.match.url}/customer`} onClick={() => setCurrentCustomerIdInRedux(customer.id)} >
-            <CustomerCard name={customer.name} amount={customer.outstandingBalance} date={getLatestReadingDate(customer)} active={customer.isactive} />
-          </Link>
-        ))
-        }
-        <FormHelperText>{filterLabels[1]}</FormHelperText>
-        {filteredCustomersAlt.map((customer, index) => (
-          <Link key={index} to={`${props.match.url}/customer`} onClick={() => setCurrentCustomerIdInRedux(customer.id)} >
-            <CustomerCard name={customer.name} amount={customer.outstandingBalance} date={getLatestReadingDate(customer)} active={customer.isactive} />
-          </Link>
-        ))
-        }
-      </BaseScrollView>
+    <BaseScreen rightIcon="user" title="Customers" searchAction={handleSearchChange} searchExit={exitSearch}>
+      <TabContext value={tabValue.toString()}>
+        <Tabs
+          textColor="primary"
+          indicatorColor="primary"
+          value={tabValue}
+          onChange={changeTab}
+          variant="scrollable"
+        >
+          <Tab className={classes.tab} label="All" value={CustomerStatus.ALL} />
+          <Tab className={classes.tab} label={getMeterTabLabel()} value={CustomerStatus.METER} />
+          <Tab className={classes.tab} label={getPaymentTabLabel()} value={CustomerStatus.PAYMENT} />
+          <Tab className={classes.tab} label="Done" value={CustomerStatus.DONE} />
+        </Tabs>
+        <div className={classes.activeContainer}>
+          <div className={classes.active}></div>
```

Comment:
This colored div thing is super jank haha could you replace this with a circle icon? `FiberManualRecord` might work.